plugins/terraform.ts

```diff
@@ -0,0 +1,79 @@
+import { spawn } from "../src/helpers.ts";
+import Plugin from "../src/plugin.ts";
+
+class Terraform implements Plugin {
+  name = "terraform";
+  commands: Record<string, (params: string[]) => Promise<void>>;
+  constructor() {
+    this.commands = {
+      init: (args: string[]) => spawn(this.name, ["init", ...args]),
+      validate: (args: string[]) => spawn(this.name, ["validate", ...args]),
+      plan: (args: string[]) => spawn(this.name, ["plan", ...args]),
+      apply: (args: string[]) => spawn(this.name, ["apply", ...args]),
+      destroy: (args: string[]) => spawn(this.name, ["destory", ...args]),
```

Comment:
There is a mistake here : `destory` -> `destroy`

Reply:
Oh, let me fix it